skins/oasis/js/connectionType.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    'use strict';
+
+    var connection = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
+
+    if (connection && connection.effectiveType) {
+        Wikia.Tracker.track({
+            action: Wikia.Tracker.ACTIONS.VIEW,
+            category: 'connection-type',
+            label: Wikia.getConnectionType(),
```

Comment:
Where is this function defined? :D

Reply:
Forgot to change after refactor, thanks for catch :)